NBitcoin.Tests/NBitcoin.Tests.csproj

```diff
@@ -4,9 +4,10 @@
 		<Company>Metaco SA</Company>
 		<Copyright>Copyright © Metaco SA 2017</Copyright>
 		<Description>The C# Bitcoin Library</Description>
+		<LangVersion>8.0</LangVersion>
 	</PropertyGroup>
   <PropertyGroup>
-	  <TargetFrameworks>net461;netcoreapp3.0</TargetFrameworks>
+	  <TargetFrameworks>netcoreapp3.0</TargetFrameworks>

```

Comment:
why remove net461?

Reply:
Oh f**k, it is because i cannot run the tests when `net461` is in the project and then I remove it again and again and again but this time I commit it by mistake. Good catch and thank you.